common/interfaces/user_interface/ui_text_slideshow.c

```diff
@@ -134,19 +132,15 @@ static void event_handler(lv_obj_t *obj, const lv_event_t event) {
   ASSERT(obj != NULL);
```

Comment:
remove this check for consistency with other UI components.

Reply:
removed